src/postgres-client/src/lib.rs

```diff
@@ -238,7 +238,7 @@ impl PostgresClient {
     fn status_metrics(&self, status: Status) {
         self.metrics
             .connpool_available
-            .set(f64::cast_lossy(status.available));
+            .set(i64::cast_from(status.available));
```

Comment:
@danhhz I changed this from a `Gauge` to an `IntGauge` because that seems to make more sense, but you might have opinions

Reply:
@bkirwi as the person who added this: it calls it being a float out in the commit comment. Was there a reason for choosing that over an `i64` / `isize`?